docs/learn/expert-guides/universal-profile/register-lsp12-issued-assets.md

```diff
@@ -178,21 +195,71 @@ const { keys: lsp12DataKeys, values: lsp12Values } = erc725.encodeData([
 ]);
 ```
 
-## Instantiate the Universal Profile contract
+  </TabItem>
+  <TabItem value="update-element" label="Add, Update, or Remove Issued Assets">
 
-Create an instance of the Universal Profile contract as shown below:
+:::success Version Compatibility
 
-<Tabs groupId="web3-lib">
-  <TabItem value="web3js" label="web3.js">
+Updating a subset of storage elements in an array requires dynamic encoding, which is only available for [`erc725.js`](../../../tools/erc725js/getting-started.md) of version 0.24.0 and above.
```

Comment:
I don't think we need to mention this. People will always the latest version of the _erc725.js_ library. Suggest removing